{"timestamp":"2025-06-15T06:43:20.983635","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-06-15T06:43:20.984413","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/postgres_connecter.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-06-15T06:43:21.294506Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.295750Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.297045Z","level":"info","event":"Current task name:insert_data_into_table","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.298174Z","level":"info","event":"Dag name:postrgres-connecter","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.031997","level":"info","event":"Executing: \n            INSERT INTO Orders VALUES (21, 'delivered')\n        ","logger":"airflow.task.operators.airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator"}
{"timestamp":"2025-06-15T06:43:21.295108","level":"info","event":"Connection Retrieved 'postgres_airflow_conn'","logger":"airflow.hooks.base"}
{"timestamp":"2025-06-15T06:43:21.309537","level":"info","event":"Connection Retrieved 'postgres_airflow_conn'","logger":"airflow.hooks.base"}
{"timestamp":"2025-06-15T06:43:21.324198","level":"info","event":"Running statement: \n            INSERT INTO Orders VALUES (21, 'delivered')\n        , parameters: None","logger":"airflow.task.hooks.airflow.providers.postgres.hooks.postgres.PostgresHook"}
{"timestamp":"2025-06-15T06:43:21.330880","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"UniqueViolation","exc_value":"duplicate key value violates unique constraint \"orders_pkey\"\nDETAIL:  Key (orderid)=(21) already exists.\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":867,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1159,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py","lineno":312,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":771,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":819,"name":"_run_command"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-06-15T06:43:21.379854Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.381615Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.382706Z","level":"info","event":"Task:<Task(SQLExecuteQueryOperator): insert_data_into_table>","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.383458Z","level":"info","event":"Failure caused by duplicate key value violates unique constraint \"orders_pkey\"","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.384200Z","level":"info","event":"DETAIL:  Key (orderid)=(21) already exists.","chan":"stdout","logger":"task"}
{"timestamp":"2025-06-15T06:43:21.384906Z","level":"info","event":"","chan":"stdout","logger":"task"}
